templates/repo/pulls/status.tmpl

```diff
@@ -1,34 +1,54 @@
-{{if $.LatestCommitStatus}}
-	{{if not $.Issue.PullRequest.HasMerged}}
-		<div class="ui top attached header">
-			{{if eq .LatestCommitStatus.State "pending"}}
-				{{$.locale.Tr "repo.pulls.status_checking"}}
-			{{else if eq .LatestCommitStatus.State "success"}}
-				{{$.locale.Tr "repo.pulls.status_checks_success"}}
-			{{else if eq .LatestCommitStatus.State "warning"}}
-				{{$.locale.Tr "repo.pulls.status_checks_warning"}}
-			{{else if eq .LatestCommitStatus.State "failure"}}
-				{{$.locale.Tr "repo.pulls.status_checks_failure"}}
-			{{else if eq .LatestCommitStatus.State "error"}}
-				{{$.locale.Tr "repo.pulls.status_checks_error"}}
-			{{else}}
-				{{$.locale.Tr "repo.pulls.status_checking"}}
-			{{end}}
-			</div>
-	{{end}}
+{{/*
+Template Attributes:
+* locale: passed through for localization
+* CommitStatus: summary of all commit status state
+* CommitStatuses: all commit status elements
+* IsTippy: whether this template is the content of tippy
```

Comment:
Shouldn't that be `IsPopup`?

Reply:
It is a popup and using tippy.js. Maybe we will convert it into other libs, so using popup is better?

Reply:
Don't think we will find anything better than tippy, so I'm okay with it.

Reply:
Ah I see you renamed, ok as well.